src/Kconfig

```diff
@@ -4,12 +4,13 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-config GOLIOTH_FW_UPDATE
-    bool "Golioth Firmware Update service"
+config GOLIOTH_OTA
+    bool "Golioth Over-the-Air Update Service"
     help
-        Enable the Golioth Firmware Update service
+        Enable the Golioth OTA service to manage the transfer of OTA components between server and
+        device. See also: GOLIOTH_FW_UPDATE
 
-if GOLIOTH_FW_UPDATE
+if GOLIOTH_OTA
 
 config GOLIOTH_OTA_THREAD_STACK_SIZE
```

Comment:
This is actually used by the fw_update module, not OTA. We should have it guarded by `GOLIOTH_FW_UPDATE` instead and also rename it.